open_academy/i18n/es_MX.po

```diff
@@ -0,0 +1,250 @@
+# Translation of Odoo Server.
+# This file contains the translation of the following modules:
+# 	* open_academy
+#
+msgid ""
+msgstr ""
+"Project-Id-Version: Odoo Server 15.0+e\n"
+"Report-Msgid-Bugs-To: \n"
+"POT-Creation-Date: 2022-10-12 18:13+0000\n"
+"PO-Revision-Date: 2022-10-12 18:13+0000\n"
+"Last-Translator: \n"
+"Language-Team: \n"
+"MIME-Version: 1.0\n"
+"Content-Type: text/plain; charset=UTF-8\n"
+"Content-Transfer-Encoding: \n"
+"Plural-Forms: \n"
+
+#. module: open_academy
+#: model:ir.model.fields,field_description:open_academy.field_openacademy_session__active
+msgid "Active"
+msgstr ""
+
+#. module: open_academy
+#: model:ir.actions.act_window,name:open_academy.open_academy_wizard_action_lunch
+#: model_terms:ir.ui.view,arch_db:open_academy.open_academy_wizard_view_form
+msgid "Add attendees"
+msgstr ""
```

Comment:
You didn't add any translation. We need to add the Spanish translation of term in `msgid` in `msgstr`
```suggestion
msgid "Add attendees"
msgstr "Añadir asistentes"
```

Also report and dashboard terms are missing here. Please  update your module and regenerate the translations.